dokku

```diff
@@ -137,7 +137,9 @@ case "$1" in
 
         if [[ -z "$DOKKU_HEROKUISH" ]]; then
           DOKKU_DOCKERFILE_PORT=$(dokku config:get $APP DOKKU_DOCKERFILE_PORT || true)
-          START_CMD=$(dokku config:get $APP DOKKU_DOCKERFILE_START_CMD || $START_CMD)
+          DOKKU_DOCKERFILE_START_CMD=$(dokku config:get $APP DOKKU_DOCKERFILE_START_CMD)
```

Comment:
@josegonzalez mentioned switching this to `config_get`. however, the main `dokku` script hasn't been refactored for this in mind and the `dokku config:get` pattern is all over the place. I vote we don't include that change in this PR.